api/models/Subjects.js

```diff
@@ -16,6 +16,12 @@ const model = {
     cid_id: {
       type: "integer",
       required: true
+    },
```

Comment:
I think we should have a separate entity for describing the detainee that the irc's talk about- let's discuss this